### PR TITLE
Adding warnings about disabling password login to rpi.md

### PR DESCRIPTION
**Pull request by dougtakeuti**

### CODE REVIEW

File: docs/docs/walkthrough/phase-0/rpi.md [hunk 1]
```diff
@@ -205,11 +205,11 @@ Run
...
 `sudo raspi-config` 
 
-to expand filesystem, change user password and set timezone (in internationalization options). This will take effect on the next reboot, so go ahead and reboot if prompted, or run `sudo reboot` when you're ready.
+Here you can expand filesystem to maximize memory, change user password and set timezone (in internationalization options). This will take effect on the next reboot, so go ahead and reboot if prompted, or run `sudo reboot` when you're ready.
 
-## Setup Password-less Login [optional]
+## Setting up an SSH key for Password-less Login [optional - WARNING: THIS COULD POTENTIALLY LOCK YOU OUT OF YOUR RASPBERRY PI]
 
-We will now setup a public/private key identity, and configure your local computer and the Raspberry Pi to automatically use it, in order to allow convenient future ssh access to the Pi without requiring a password.
+You can setup a public/private key identity, and configure your local computer and the Raspberry Pi to automatically use it. If you continue with the configuration and Disable Password Login (next step), this will allow SSH access to the Pi without requiring a password. Some people find this feature very convenient.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-0/rpi.md]: Passwordless login works great even if you don't disable password login. Can probably remove the "If you continue" clause here.

File: docs/docs/walkthrough/phase-0/rpi.md [hunk 2]
```diff
@@ -241,12 +241,22 @@ Instead of appending it to the list of authorized keys, you may simply copy your
...
+### Disabling password login [optional - WARNING: THIS COULD POTENTIALLY LOCK YOU OUT OF YOUR RASPBERRY PI]
+Make sure you fully understand this feature before proceeding. It is completely optional. Some people think that logging in to the Pi without typing a password is a real time saver and also more secure. There are two ways of securing the Pi:
+
+1) Setting a password: 
+
+	a) use `sudo raspi-config` as described above, or 
+	
+	b) `sudo passwd`
+
+2) Disabling password login completely. In this case, you can ONLY log in with your SSH key. Be carefull here.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-0/rpi.md]: "Careful"

**Comment by scottleibrand** [comment]
Thanks. Two little things I noted inline. Feel free to fix those before we merge, or we can do so later.

**Comment by dougtakeuti** [comment]
Is it better now?

2016-05-21 1:29 GMT-03:00 Scott Leibrand <notifications@github.com>:

**Pull request merged**